tests/utils_tests/test_timezone.py

```diff
@@ -12,14 +12,17 @@
     except ImportError:
         zoneinfo = None
 
-from django.test import SimpleTestCase, override_settings
+from django.test import SimpleTestCase, ignore_warnings, override_settings
 from django.utils import timezone
+from django.utils.deprecation import RemovedInDjango50Warning
 
 CET = pytz.timezone("Europe/Paris")
 EAT = timezone.get_fixed_timezone(180)      # Africa/Nairobi
 ICT = timezone.get_fixed_timezone(420)      # Asia/Bangkok
 UTC = datetime.timezone.utc
 
+# TODO: Remove the zoneinfo = None fallback and these helpers.
+#   ???: for 4.0 or 5.0?
```

Comment:
I'll put my vote in for 4.0 as we're making `zoneinfo` the default with `pytz` as the backward compatible option that you have to opt in to. Also were not falling back to `zoneinfo = None` in `django.utils.timezone`.